sarracenia/sr.py

```diff
@@ -1608,15 +1608,25 @@ def disable(self):
                 logging.error("cannot disable %s while it is running! " % f)
                 continue
 
-            state_file_cfg = self.user_cache_dir + os.sep + c + os.sep + cfg
-            state_file_cfg_disabled = state_file_cfg + os.sep + 'disabled'
-            if os.path.exists(state_file_cfg_disabled):
+            if self.configs[c][cfg]['options'].statehost:
+                state_file_dir = self.user_cache_dir + os.sep + self.hostdir + os.sep + f.replace('/', os.sep)
+            else:
+                state_file_dir = self.user_cache_dir + os.sep + f.replace('/', os.sep)
+
+            logger.critical( f"{state_file_dir=}" )
```

Comment:
i think this is probably a leftover from debugging? `logger.critical( f"{state_file_dir=}" )` :)

Reply:
there's another one on line 1621 as well. 

Otherwise I think it's good

Reply:
nice catch... those will break on 18.04... 
will fix.